### ISSUE TITLE
[BUG] [Linter] false positives for orphan tags when django templating is used

### ISSUE THREAD

**Issue by thacoon** [djlint/djLint#874]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: macOS Sonoma
- Python Version 3.11.9
- djLint Version 1.34.1
- template language: django default templating language

## Issue
djlint reports false positives orphan tags for a snippet in which django templating is used. If I remove the django templating no more orphan tags are reported.

## How To Reproduce

Detects two orphan tags for this snippet:
```html
{% load tailwind_filters %}

<div class="relative flex items-center">
  <div class="flex h-6 items-center">
    <input id="{{ field.id_for_label }}" name="{{ field.html_name }}" {% if field.value %}checked{% endif %} type="checkbox" class="h-5 w-5 rounded border-gray-300 text-primary-black focus:ring-primary-black" {{ field|build_attrs }} />
  </div>
  <label for="{{ field.id_for_label }}" class="ml-3 text-sm font-normal text-typography-default">
    {{ field.label|safe }}
    {% if field.field.required %}<span>*</span>{% endif %}
  </label>
</div>
```

For this snippet it displays the following error:
```
H025 3:0 Tag seems to be an orphan. <div class="relative
H025 11:2 Tag seems to be an orphan. </label>
```

No orphan tags are reported when django templating is not used:
```html
<div class="relative flex items-center">
  <div class="flex h-6 items-center">
    <input type="checkbox" class="h-5 w-5 rounded border-gray-300 text-primary-black focus:ring-primary-black" />
  </div>
  <label class="ml-3 text-sm font-normal text-typography-default">
    Label
    <span>*</span>
  </label>
</div>
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
```toml
[tool.djlint]
ignore = "H006,H030,H031"
include = "H017,H035"
indent = 2
blank_line_after_tag = "load,extends"
profile = "django"
max_line_length = 120
format_attrite_template_tags = true
```

**Comment by monosans** [comment]
I can't reproduce it. It's probably fixed. Let me know if I'm wrong.
